src/mlpack/methods/local_coordinate_coding/local_coordinate_coding_main.cpp

```diff
@@ -149,6 +149,22 @@ static void mlpackMain()
         matX.col(i) /= norm(matX.col(i), 2);
     }
 
+    if (CLI::GetParam<int>("atoms") < 1)
+    {
+      Log::Fatal << "The number of atoms should be atleast one!" << endl;
+    }
```

Comment:
Can you use the functions in `src/mlpack/core/util/param_checks.hpp` for these three parameter checks?  Example:

```
RequireParamValue<int>("atoms", [](int x) { return x > 0; },
    "number of atoms must be greater than 1");
```